src/sentry/utils/suspect_resolutions/commit_correlation.py

```diff
@@ -25,8 +23,13 @@ class ReleaseCommitFileChanges:
 def is_issue_commit_correlated(
     resolved_issue: int, candidate_issue: int, project: int
```

Comment:
Don't have to change this now, but is there a reason why `resolved_issue` and `candidate_issue` couldn't be of type `Group` instead of int here? Same for why `project` isn't a `Project` but an int. 

If this function is passed the `Group` and `Project`, that's one less thing for this function to worry about. :)